node/src/BaseClient.ts

```diff
@@ -1927,6 +1943,7 @@ export class BaseClient {
      * @remarks Since Valkey version 6.2.0.
      *
      * @param key - The key of the hash.
+     * @param options - (Optional) Parameter to change encoding of returned object. See {@link DecoderOption}.
```

Comment:
You can shorten this
```suggestion
     * @param options - (Optional) See {@link DecoderOption}.
```